src/main/java/core/basesyntax/Machines/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.Machines;
+
+public class Truck extends Machine{
```

Comment:
Please add a space between the superclass `Machine` and the curly brace to follow Java code conventions.